### DISCUSSION TITLE
modded minecraft fabric problem

### DISCUSSION THREAD

**Discussion by wackoamd1** [GameServerManagers/LinuxGSM#3937] [Q&A] [answered]
hi all 
this whole linux stuff is new to me

i have a problem with minecraft fabric ,

i installed minecraft java with linuxgsm , works fine
i dropped the fabric loader in it (made the changes in config , works fine ( makes the extra directories creates the world all fine.

now here is my problem
i have a running fabric server on truenas , which im trying to get onto the linux server.
assoon as i drop the mods from the fabric truenas server into the mods folder of this linux server , it will not start the server anymore.
 sp basicly what im asking , what could be wrong if i just move a working minecraft server install to the linux server that it doesnt work anymore.
thanks in advance

**Comment by johnoclockdk** [comment, marked as answer]
you need to get fabric jar and drop it in the mod folders

**Reply by StormGaming** [reply]
Can you go into more detail for this please? I am having the same issue and I don't know what "fabric jar" you are refering to. Sorry for the late reply

**Reply by johnoclockdk** [reply]
https://fabricmc.net/use/server/ download you needed version and drop in folder

**Comment by StormGaming** [comment]
Hi, thanks for that. Turns out I had everything set up already but I just
needed to start the server with admin privileges and then it works fine.

Thanks for the reply though!
